services/QuillLMS/app/models/auth_credential.rb

```diff
@@ -28,7 +28,11 @@ class AuthCredential < ApplicationRecord
   belongs_to :user
 
   GOOGLE_PROVIDER = 'google'
-  EXPIRATION_DURATION = 6.months
+  GOOGLE_EXPIRATION_DURATION = 6.months
+
+  CLEVER_DISTRICT_PROVIDER = 'clever_district'
+  CLEVER_LIBRARY_PROVIDER = 'clever_library'
+  CLEVER_EXPIRATION_DURATION = 24.hours
```

Comment:
Take or leave, but if we are expiring the credential in `23.hours`, you could consider just storing that here for simplicity.

Reply:
Having to keep track of the offset elsewhere is error-prone so I definitely like this idea.